tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "target": "esnext",
```

Comment:
I would personally favor target: "es5" to specify that we expect to generate ES5 code (using a JavaScript transpiler).

Reply:
Sure 
Just let me wakeup ^^ I will change it in few hours